persist/dynamo/persister_test.go

```diff
@@ -0,0 +1,140 @@
+package dynamo_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/ourrootsorg/cms-server/model"
+	"github.com/ourrootsorg/cms-server/persist/dynamo"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCategory(t *testing.T) {
+	table := os.Getenv("DYNAMODB_TEST_TABLE_NAME")
```

Comment:
Could you add a condition to skip this test if testing.Short()?

Reply:
and also the other tests in this file?

Reply:
Done.